Source/ProgressionSystemRuntime/Public/Components/PSSpotComponent.h

```diff
@@ -58,4 +66,8 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSSpotComponent : public UActorComponent
 	/** Updates the progression unlocked skins when score changes */
 	UFUNCTION(BlueprintNativeEvent, Category= "C++", meta = (BlueprintProtected))
 	void OnCurrentScoreChanged(const FPSSaveToDiskData& CurrentSaveToDiskDataRow, const FPSRowData& CurrentProgressionSettingsRow);
+	
+	/** Check is player is allowed to play with current skin if not switch to allowed */
+	UFUNCTION(BlueprintCallable, Category= "C++")
```

Comment:
- new function is under `protected` , so please also add `BlueprintProtected` meta to the function itself, otherwise it will be still accessible in other blueprint actors
- please perform file reformat (ctrl+S) as this function is added with extra tab above. Extra tabs I also can see above BeginPlay

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/63/commits/5c87646deb1070100ec954abf744911df2a210b8